src/mc3.h

```diff
@@ -0,0 +1,37 @@
+#ifndef MC3
```

Comment:
I think this file should already be in dev_branch? This surprised me (I'll check, you don't have to do anything but I want this to be written down somewhere)

Reply:
Ok I've fixed this